### ISSUE TITLE
Add a distribution folder with minified version and sourcemap

### ISSUE THREAD

**Issue by Mintonne** [x2js/x2js#80]
||**Unminified**|**Minified**|**Compression**
:-----:|:-----:|:-----:|:-----:
File size|~23.44 KB|~8.01 KB|~65%
Gzipped|~7.17 KB|~3.55 KB|~50%

Test here - https://mintonne.github.io/x2js/

![annotation 2019-01-10 032435](https://user-images.githubusercontent.com/39894801/50938525-7d254d00-1470-11e9-8a6d-14b280ebb26c.jpg)

I have an [uglify-js](https://www.npmjs.com/package/uglify-js) script that creates the minified script and sourcemap.

```javascript
"dist": "rm -rf dist && mkdir dist && npx uglifyjs x2js.js -o dist\\x2js.min.js -c -m --source-map \"includeSources='true',url='x2js.min.js.map'\""
```

**Comment by Maxouhell** [comment]
I'll check this ASAP (probably on monday)

**Comment by Mintonne** [comment]
Hey @Maxouhell,

Any update on this?

**Comment by Maxouhell** [comment]
Yep ! 
I will publish the Typescript version tomorrow. 

Sorry for the delay, I had a problem with ES5 bundle.

**Comment by Maxouhell** [comment]
Hi !
I didn't find out why there is a problem with the bundle...

If I didn't find it out before, tomorrow I will publish a version with uglifyjs, so you can work before I'm solving this.

**Comment by Maxouhell** [comment]
Ok, 
According on your fork, I've published the dist/min files in the v3.2.5.

I can't publish the Typescript version for now, but I'll fix it as soon as possible, so expect to see another version happen in the next days.
